threads.go

```diff
@@ -541,11 +551,24 @@ func (t *threads) Subscribe(opts ...options.SubOption) tserv.Subscription {
 
 // putRecord adds an existing record. See PutOption for more.
 func (t *threads) putRecord(ctx context.Context, id thread.ID, lid peer.ID, rec thread.Record) error {
```

Comment:
Ok, main purpose of the PR. Putting a new record when also fetching all missing history up to the current head. And notify in bus in order of all added things.